docs/api-docs/store-management/shipping/shipping-provider-api.md

```diff
@@ -65,25 +112,25 @@ BigCommerce will send requests to your server to get information back about ship
 
 Since BigCommerce will be sending requests, you will need to provide BigCommerce with URLs that can accept quote requests and optionally a URL to check and validate connection options during app registration. These can be any valid HTTPS URL using port `443`, for example:

```

Comment:
```suggestion
Because BigCommerce sends requests, you need to provide BigCommerce with URLs that accept quote requests and, optionally, a URL to check and validate connection options during app registration. These can be any valid HTTPS URL using port `443`, for example:

```

Reply:
better than what was there.  adopted & revised further.